shared/constants/team-building.tsx

```diff
@@ -79,7 +79,14 @@ const parseRawResultToUser = (
       id: result.contact.assertion,
       label: result.contact.displayLabel,
       prettyName: result.contact.displayName,
-      serviceMap: {...serviceMap, keybase: result.contact.username},
+      serviceMap: {keybase: result.contact.username},
```

Comment:
no `...serviceMap` anymore?

Reply:
It was always empty for contacts, but I guess it's ok to keep it so it magically works once we bring this in on core side